### PR TITLE
Update files to be compatible with CF/CFI

### PR DESCRIPTION
**Pull request by piyush-J**

### CODE REVIEW

File: src/universe/GUTChecker.java [hunk 1]
```diff
@@ -1,17 +1,12 @@
 package universe;
 
+import checkers.inference.*;
```
Comment (piyush-J) [inline src/universe/GUTChecker.java]: Don't use this * import

File: src/universe/GUTVisitor.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 import org.checkerframework.common.basetype.BaseAnnotatedTypeFactory;
 import org.checkerframework.common.basetype.BaseTypeValidator;
 import org.checkerframework.common.basetype.BaseTypeVisitor;
-import org.checkerframework.framework.source.Result;
+// import org.checkerframework.framework.source.DiagMessage;
```
Comment (piyush-J) [inline src/universe/GUTVisitor.java]: Remove

File: src/universe/GUTVisitor.java [hunk 3]
```diff
@@ -158,6 +159,10 @@ protected boolean checkReceiverOverride() {
         };
     }
 
+    @Override
```
Comment (piyush-J) [inline src/universe/GUTVisitor.java:156]: Add the reason for this

**Comment by piyush-J** [comment]
The test directory will be moved to src in a different PR

File: src/universe/GUTAnnotatedTypeFactory.java [hunk 4]
```diff
@@ -107,6 +108,17 @@ public void addComputedTypeAnnotations(Element elt, AnnotatedTypeMirror type) {
         super.addComputedTypeAnnotations(elt, type);
     }
 
+    /**
+     * Replace annotation of extends or implements clause with SELF in GUT.
+     */
+
```
Comment (wmdietl) [inline src/universe/GUTAnnotatedTypeFactory.java]:
```suggestion
```

**Pull request merged**